tensorflow/python/lib/core/py_func.h

```diff
@@ -16,6 +16,12 @@ limitations under the License.
 #ifndef TENSORFLOW_PYTHON_LIB_CORE_PY_FUNC_H_
 #define TENSORFLOW_PYTHON_LIB_CORE_PY_FUNC_H_
 
+#if NUMPY_IMPORT_ARRAY_RETVAL == NULL
```

Comment:
Can you add a comment to explain why this is necessary? Something like "The return type of import_array() changed between Python 2 and 3"